### PR TITLE
feat: Added languages to dropdown in settings

### LINKED ISSUE #4737: Localized languages should appear in dropdown in settings
Per the localization pipeline into 17 Azure languages.

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Description

The 17 supported languages need to be in the dropdown.

## Task Item

Fixed #4737 

## Screenshots

image

### CODE REVIEW

File: Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx [hunk 1]
```diff
@@ -50,15 +50,37 @@ const AppSettings: React.FC<RouteComponentProps> = () => {
...
+    { key: 'ko', text: '한국어‎', englishName: 'Korean' },
+    { key: 'nl', text: 'Nederlands‎', englishName: 'Dutch (Netherlands)' },
+    { key: 'pl', text: 'Polski‎', englishName: 'Polish' },
+    { key: 'pt-BR', text: 'Português (Brasil)‎', englishName: 'Portuguese (Brazil)' },
+    { key: 'pt-PT', text: 'Português (Portugal)‎', englishName: 'Portuguese (Portugal)' },
+    { key: 'ru', text: 'Русский‎', englishName: 'Russian' },
+    { key: 'sv', text: 'Svenska‎', englishName: 'Swedish' },
+    { key: 'tr', text: 'Türkçe‎', englishName: 'Turkish' },
+    { key: 'zh-Hans', text: '中文(简体)‎', englishName: 'Chinese(Simplified)' },
+    { key: 'hz-Hant', text: '中文(繁體)', englishName: 'Chinese(Traditional)' },
```
Comment (beyackle) [inline Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx:71]: Space before the parens here

**Review by beyackle — CHANGES_REQUESTED** [review]
Looks good, just one tiny change needed.

File: Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx (hunk 1 above)
Comment (GeoffCoxMSFT) [reply Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx:71]: The Azure doesn't have a space, so I would like to leave as is to match.

Comment (boydc2014) [reply Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx:71]: it should be "zh-Hant" other than "hz-Hant"

File: Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx [hunk 2]
```diff
@@ -50,15 +50,37 @@ const AppSettings: React.FC<RouteComponentProps> = () => {
...
   if (process.env.NODE_ENV !== 'production') {
     languageOptions.push({
       key: 'en-US-pseudo',
       text: formatMessage('Pseudo'),
+      englishName: 'pseudo',
     });
     languageOptions.push({
       key: 'en-US-DoesNotExist',
       text: formatMessage('Does Not Exist'),
+      englishName: 'does not exist',
```
Comment (boydc2014) [inline Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx:83]: why do we need a "Does Not Exits"?

Comment (GeoffCoxMSFT) [reply Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx:83]: I think does not exist is there for testing when the language selected is not found.  Pseudo and Does Not Exist only show up in dev mode.

File: Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx (hunk 1 above)
Comment (GeoffCoxMSFT) [reply Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx]: Great catch on the zh-Hant typo!

File: Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx [hunk 3]
```diff
@@ -50,15 +50,37 @@ const AppSettings: React.FC<RouteComponentProps> = () => {
 
   const renderElectronSettings = isElectron();
 
-  const languageOptions = [{ key: 'en-US', text: formatMessage('English (US)') }];
+  const languageOptions = [
```
Comment (hatpick) [inline Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx:53]: This can be moved to the outside of the component code.

**Review by beyackle — DISMISSED** [review]

**Review by beyackle — APPROVED** [review]

**Pull request merged**